.expo/README.md

```diff
@@ -0,0 +1,15 @@
+> Why do I have a folder named ".expo" in my project?
```

Comment:
Can you please add instructions for how you can run this example project?

Reply:
I will create a separate issue for this.